app/serializers/rss/account_serializer.rb

```diff
@@ -5,12 +5,12 @@ class RSS::AccountSerializer
   include StreamEntriesHelper
   include RoutingHelper
 
-  def render(account, statuses)
+  def render(account, statuses, tag)
     builder = RSSBuilder.new
 
     builder.title("#{display_name(account)} (@#{account.local_username_and_domain})")
            .description(account_description(account))
-           .link(TagManager.instance.url_for(account))
+           .link("#{TagManager.instance.url_for(account)}#{Addressable::URI.parse("/tagged/#{tag}").normalize if tag.present?}")
```

Comment:
Since `RoutingHelper` is already included, it is probably better to use route helpers:

```ruby
.link(tag.present? ? short_account_tag_url(account, tag) : short_account_url(account))
```